scripts/build-commonjs.sh

```diff
@@ -2,14 +2,7 @@
 
 npx tsc --module commonjs --target es5 --outDir dist/commonjs
 
-SED_OPTIONS=
```

Comment:
The replace inline `-i ''` sed argument for macOS added '' to the files, instead of leaving the file names untouched. New approach is os independent, rendering the `$SED_OPTIONS` redundant.